content/blog/leorize/2020-08-31_gsoc_2020_final_report_improving_and_extending_services_kit.md

```diff
@@ -0,0 +1,61 @@
++++
+type = "blog"
+title = "GSoC 2020 Final Report: Improving and Extending Services Kit"
+author = "leorize"
+date = "2020-08-31 02:00:18-05:00"
+tags = ["haiku", "software"]
```

Comment:
these are rather useless tags on our website (we could tag everything this way). maybe tag with gsoc and gsoc2020?